### ISSUE TITLE
Issues launching Power Actions from vCenter web UI

### ISSUE THREAD

**Issue by Nag5454** [vmware/script-runtime-service-for-vsphere#83]
### Describe the bug

We have been dealing with some issues launching VMware Power Actions from vCenter web client. We got the latest version installed and configured. Not sure what exactly is the problem here. The error we see from vCenter is provided below. Unfortunately, we are unable to open a ticket with VMware support either.

 Error retrieving the console.
A general error occurred. Please check the logs for more information.

### Reproduction steps

1. Deploy VMware power actions appliance using the .OVA file
2. Configure a static IP, DNS etc. 
3. Launch the Power Actions by going into vSphere - Developer Center
...

### Expected behavior

We expected the Power Actions to launch the PowerCLI client from the vCenter itself. Instead, we are seeing the below error.

 Error retrieving the console.
A general error occurred. Please check the logs for more information.

**Comment by kamennikolov** [comment]
Is this only when you try to open the console or is there the same issue when trying to open the other two windows (script library and script runs) as well?

**Comment by Nag5454** [comment]
It's the same issue no matter what we do with the power actions.


Thanks,
Nag
________________________________

**Comment by NedyalkaD** [comment]
Hi Nag, can you, please upload logs so that we can debug the issue.

**Comment by Nag5454** [comment]
Is there an instruction document on how to collect the logs?

**Comment by kamennikolov** [comment]
Check the "Troubleshooting Power Actions" chapter in the Power Actions user's guide here: https://vdc-download.vmware.com/vmwb-repository/dcr-public/b89e20ba-ad9c-424a-ae59-c40b8d4eafef/52ca1971-0791-4ab8-82b3-c4481d8b0671/Power%20Actions%20User%27s%20Guide.pdf

**Comment by gungazoo** [comment]
I have the same issue. In the 'poweractions-ui.log' log I see these three messages about unable to verify the STS certificate.

```
2024-01-08 21:00:04.642  INFO 18 --- [nio-8443-exec-3] c.v.v.s.c.impl.ssl.StsSslTrustManager    : creating keystore of type jks

2024-01-08 21:00:04.643  INFO 18 --- [nio-8443-exec-3] c.v.v.s.c.impl.ssl.StsSslTrustManager    : using PKIX algorithm to create TrustManagerFactory

2024-01-08 21:00:04.688 ERROR 18 --- [nio-8443-exec-3] c.v.vim.sso.client.impl.SoapBindingImpl  : The SSL certificate of STS service cannot be verified
```

The vCenter does have a self-signed and cert verification was disabled.

**Comment by SergeiShestopalov** [comment]
I have the same issue. 
During the test, it turned out that the problem occurs with vCenter, on which a Machine SSL certificate is installed, signed by a Custom Certificate Authority, and third-party root certificates installed.

**Comment by kamennikolov** [comment]
@ShestopalovSergey yes, we were also able to reproduce the issue by changing the machine certificate. We're working on a fix.
